tests/baselines/reference/truthinessCallExpressionCoercion2.errors.txt

```diff
@@ -81,6 +83,16 @@ tests/cases/compiler/truthinessCallExpressionCoercion2.ts(112,14): error TS2774:
         if (((required1 && b))) {
             required1();
         }
+    
+        // error, extra parens are on purpose here
+        if ((required1)) {}
+             ~~~~~~~~~
+!!! error TS2774: This condition will always return true since this function is always defined. Did you mean to call it instead?
+    
+        // error
+        if (b && (required1 || required2)) {}

```

Comment:
Error should also be raised for `required2` here - I need to dig deeper to figure out a fix for it though

Reply:
I've managed to make it work.